boefjes/boefjes/plugins/kat_maxmind_geoip/main.py

```diff
@@ -0,0 +1,122 @@
+import hashlib
+import io
+import json
+import os
+import re
+import shutil
+import tarfile
+from datetime import datetime, timezone
+from os import getenv
+from pathlib import Path
+
+import maxminddb
+import requests
+
+from boefjes.job_models import BoefjeMeta
+
+BASE_PATH = Path(getenv("OPENKAT_CACHE_PATH", Path(__file__).parent))
+GEOIP_PATH_PATTERN = r"GeoLite2-City_\d+/GeoLite2-City.mmdb"
+GEOIP_META_PATH = BASE_PATH / "geoip-meta.json"
+GEOIP_SOURCE_URL = "https://download.maxmind.com/geoip/databases/GeoLite2-City/download?suffix=tar.gz"
+GEOIP_CACHE_TIMEOUT = 7200  # in seconds
```

Comment:
Should we make this cache valid for a day, or more? Is there a known interval for the source_url refresh?

Reply:
I think they refresh every week. Maybe its safest to change the timeout to a day as default?

Reply:
Set it to one day